packages/libs/components/src/stories/tidytree/TreeTable.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Story } from '@storybook/react';
+import TreeTable, { TreeTableProps } from '../../components/tidytree/TreeTable';
+import { MesaColumn } from '../../../../coreui/lib/components/Mesa/types';
+
+export default {
+  title: 'TreeTable',
+  component: TreeTable,
+  parameters: {},
+  argTypes: {},
+} as Meta;
+
+// the file is in the public/data directory
+const sevenLeafTree =
+  '(Bovine:0.69395,(Gibbon:0.36079,(Orang:0.33636,(Gorilla:0.17147,(Chimp:0.19268, Human:0.11927):0.08386):0.06124):0.15057):0.54939,Mouse:1.21460)';
+
+interface LeafRow {
+  leafId: string;
+  accession: string;
+  length: number;
+  domains: string;
+}
+
+const sevenLeafTableRows: LeafRow[] = [
+  {
+    leafId: 'Bovine',
+    accession: 'bov0123',
+    length: 99,
+    domains: 'ABC XYZ',
+  },
+  {
+    leafId: 'Gibbon',
+    accession: 'gib0987',
+    length: 120,
+    domains: 'ABC XYZ',
+  },
+  {
+    leafId: 'Orang',
+    accession: 'ora0333',
+    length: 102,
+    domains: 'ABC XYZ QQQ',
+  },
+  {
+    leafId: 'Gorilla',
+    accession: 'gor0321',
+    length: 119,
+    domains: 'ABC',
+  },
+  {
+    leafId: 'Chimp',
+    accession: 'chi0111',
+    length: 111,
+    domains: 'ABC XYZ',
+  },
+  {
+    leafId: 'Human',
+    accession: 'hum0008',
+    length: 104,
+    domains: 'ABC XYZ',
+  },
+  {
+    leafId: 'Mouse',
+    accession: 'mus0738',
+    length: 150,
+    domains: 'ABC DEF XYZ',
+  },
+];
+
+const tableColumns: MesaColumn<LeafRow>[] = [
```

Comment:
idk enough to know... is it easy to add a column of check boxes?

Reply:
Thanks for having a look. There's a separate story in "HorizontalDendrogram" with some non-table checkboxes that colour the tree.

I should probably wire up the TreeTable to use table-based checkboxes. Would be a good thing for the story.